### ISSUE TITLE
Problem when using import dependency on covidcast

### ISSUE THREAD

**Issue by bnaras** [cmu-delphi/covidcast#144]
The variable `state_census` is not available to `covidcast::name_to_abbr()` when the package is imported rather than attached via `library()` (reprex below). The fix is to use `R/sysdata.rda` for internal use. But it appears these datasets are exported? If they have to be so, perhaps some convenience functions for access?

```{r}
> covidcast::name_to_abbr("Penn")
Error in eval(lhs, parent, parent) : object 'state_census' not found
```

**Comment by capnrefsmmat** [comment]
Yes, `state_census` (and `county_census` and similar) is exported, since we imagine covidcast users may want the data in convenient form for other analyses.

Is it possible to use something like [this trick](https://stackoverflow.com/a/16355495) to ensure the data gets into the package's namespace, so its functions can use it, while still exporting it? It seems quite odd that R doesn't have a mechanism for datasets used both internally and exported.

**Comment by bnaras** [comment]
Yes, that trick will work for now as it is only sticking data objects in the package namespace although I rarely see it in packages. (I generally don't like to muck around with parent environments etc. in `.onLoad`.) The alternative would be to have functions for each exported data set, which is not ideal. So the trick is ok, I'd say, with some comments thrown in case problems arise in the future. The documentation for `data()` states it was designed before lazy loading became common, so it has warts.

**Comment by ryantibs** [comment]
@capnrefsmmat @bnaras Am I missing something ... isn't this fixed by just referring to the data objects within the covidcast package code itself as in `covidcast::state_census`, etc.?

Or will this create some kind of weird recursion issue?

**Comment by bnaras** [comment]
@ryantibs @capnrefsmmat I like your solution: simple and not much work. Apparently it is also CRAN approved. https://coolbutuseless.github.io/2018/12/10/r-packages-internal-and-external-data/

**Comment by bnaras** [comment]
@ryantibs @capnrefsmmat I'll make a PR in the next 10 min to fix this.

**Comment by ryantibs** [comment]
Closed by #148.
